.travis.yml

```diff
@@ -1,9 +1,25 @@
+dist: xenial
 language: python
 python:
   - "2.7"
+  - "3.6"
+
 install:
-  - pip install --upgrade pip
-  - pip install --upgrade numpy scipy matplotlib pandas
-  - pip install --upgrade scikit-learn
+  - pip install tox-travis
+
 script:
-  - python -m unittest discover -s test -p '*_test.py'
+  - tox
+
+
+# The following section deploys to pypi when a version tag is applied to master, see https://docs.travis-ci.com/user/deployment/pypi/
+deploy:
+  - provider: pypi
```

Comment:
This section won't work as-is currently, it can be leveraged to automatically publish to pypi on version tag.
It will work if we fill in the `password secure` section (travis has a way of encrypting pwds)